opencti-platform/opencti-graphql/src/modules/case/case-incident/case-incident-types.ts

```diff
@@ -0,0 +1,35 @@
+import type { BasicStoreEntity, StoreEntity } from '../../../types/store';
+import type { StixDomainObject, StixOpenctiExtensionSDO } from '../../../types/stix-common';
+import { STIX_EXT_OCTI } from '../../../types/stix-extensions';
+
+export const ENTITY_TYPE_CONTAINER_CASE_INCIDENT = 'Case-Incident';
+
+export interface BasicStoreEntityCaseIncident extends BasicStoreEntity {
+  name: string,
+  description: string,
+  context: string,
```

Comment:
Why we have context property ? (same question for feedback)

Reply:
✅